src/test/plugins/shared_dictionaries_plugin_test.cpp

```diff
@@ -0,0 +1,265 @@
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base_test.hpp"
+#include "lib/utils/plugin_test_utils.hpp"
+
+#include "../../plugins/shared_dictionaries_plugin/shared_dictionaries_column_processor.hpp"
+#include "../../plugins/shared_dictionaries_plugin/shared_dictionaries_plugin.hpp"
```

Comment:
Are relative imports indeed needed or would 
```
#include "plugins/shared_dictionaries_plugin/shared_dictionaries_column_processor.hpp"
#include "plugins/shared_dictionaries_plugin/shared_dictionaries_plugin.hpp"
```
already suffice?

Reply:
We removed the unused import, however the import does not work without the relative import.